From 2996ec02a5e442bc8edbaebb0bf0973c6ccfdd5f Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Mon, 7 Nov 2016 13:39:28 +0100
Subject: [PATCH] don't ignore return value, verify it

if we can't mkdir the output dir all tests will fail on creating the output
file, but rather implicitly at that
---
 autotests/testhighlighter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autotests/testhighlighter.cpp b/autotests/testhighlighter.cpp
index f802381..2b2118c 100644
--- a/autotests/testhighlighter.cpp
+++ b/autotests/testhighlighter.cpp
@@ -144,7 +144,7 @@ private Q_SLOTS:
                 << syntax;
         }
 
-        QDir().mkpath(QStringLiteral(TESTBUILDDIR "/output/"));
+        QVERIFY(QDir().mkpath(QStringLiteral(TESTBUILDDIR "/output/")));
     }
 
     void testHighlight()
-- 
2.7.4

